docs/standard/linq/creating-xml-trees.md

```diff
@@ -0,0 +1,425 @@
+            new XElement("Address",  
+                new XElement("Street1", "123 Main St"),  
+                new XElement("City", "Mercer Island"),  
+                new XElement("State", "WA"),  
+                new XElement("Postal", "68042")  
+            )  
+        )  
+    );  
+Console.WriteLine(contacts);  
+```  
+  
+This example produces the following output:  
+  
+```xml  
+<Contacts>  
+  <Contact>  
+    <Name>Patrick Hines</Name>  
+    <Phone>206-555-0144</Phone>  
+    <Address>  
+      <Street1>123 Main St</Street1>  
+      <City>Mercer Island</City>  
+      <State>WA</State>  
+      <Postal>68042</Postal>  
+    </Address>  
+  </Contact>  
+</Contacts>  
+```  
+
+#### Creating an XElement with an XAttribute
+If you pass an instance of the <xref:System.Xml.Linq.XAttribute> class for the content argument, the constructor creates an element with an attribute:
+
+```csharp  
+XElement phone = new XElement("Phone",  
+    new XAttribute("Type", "Home"),  
+    "555-555-5555");  
+Console.WriteLine(phone);  
+```  
+  
+This example produces the following output:  
+  
+```xml  
+<Phone Type="Home">555-555-5555</Phone>
+```   
+
+#### Creating an empty element  
+To create an empty <xref:System.Xml.Linq.XElement>, you do not pass any content to the constructor. The following example creates an empty element:  
+  
+```csharp  
+XElement n = new XElement("Customer");  
+Console.WriteLine(n);  
+```  
+  
+This example produces the following output:  
+  
+```xml  
+<Customer />  
+```  
+  
+#### Attaching vs. cloning  
+As mentioned previously, when adding <xref:System.Xml.Linq.XNode> (including <xref:System.Xml.Linq.XElement>) or <xref:System.Xml.Linq.XAttribute> objects, if the new content has no parent, the objects are simply attached to the XML tree. If the new content already is parented and is part of another XML tree, the new content is cloned, and the newly cloned content is attached to the XML tree.  
+
+The following example demonstrates the behavior when you add a parented element to a tree, and when you add an element with no parent to a tree.
+
+```csharp  
+// Create a tree with a child element.  
+XElement xmlTree1 = new XElement("Root",  
+    new XElement("Child1", 1)  
+);  
+  
+// Create an element that is not parented.  
+XElement child2 = new XElement("Child2", 2);  
+  
+// Create a tree and add Child1 and Child2 to it.  
+XElement xmlTree2 = new XElement("Root",  
+    xmlTree1.Element("Child1"),  
+    child2  
+);  
+  
+// Compare Child1 identity.  
+Console.WriteLine("Child1 was {0}",  
+    xmlTree1.Element("Child1") == xmlTree2.Element("Child1") ?  
+    "attached" : "cloned");  
+  
+// Compare Child2 identity.  
+Console.WriteLine("Child2 was {0}",  
+    child2 == xmlTree2.Element("Child2") ?  
+    "attached" : "cloned");  
+
+// The example displays the following output:  
+//    Child1 was cloned  
+//    Child2 was attached  
+```
+
+## Creating XML Trees in Visual Basic
```

Comment:
Let's split this topic in two like we used to have before (one for C# and one for VB)
https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2008/bb387100(v=vs.90)